www/src/css/banner.css

```diff
@@ -1,121 +1,37 @@
 /** @define b-banner; weak */
 
-.b-banner {
-  background-color:#37373d;
-  color:#efefef;
-  line-height:20px;
-}
-.b-banner .b-layout__container {
-  display:flex;
-  align-items:center;
-}
-
-.b-banner img {
-  display:block;
-}
-
-.b-banner a {
-  text-decoration:none;
-  color:#fff;
-  display:block;
-}
-
-.b-banner a.active {
-}
-
-.b-banner__brand a {
-  display: flex;
-  align-items:center;
-  padding:10px 1em;
-}
-
-.b-banner__brand img {
-  margin:0 10px 0 0;
-}
-
-.b-banner__search {
-  flex:1;
-}
-
-.b-banner__form {
-  display:flex;
-}
-
-.b-banner__search input[type=search] {
-  width:50%;
-  border:none;
-  height:24px;
-  line-height:20px;
-  padding:12px 25px 12px 15px;
-  background-color:#333;
-}
-
-.b-banner__search input[type=search]:focus {
-  outline:0;
-}
-
-.b-banner__submit {
-  display:block;
-  border:0;
-  height:60px;
-  width:60px;
-  background: #333 url(/public/images/search.svg) 50% 50% no-repeat;
-  background-size:20px;
-  cursor:pointer;
-}
-
-.b-banner__submit:active, 
-.b-banner__submit:focus {
-  outline:0;
-}
-
-.b-banner__submit:hover {
-  background-color:#dd4814;
+.b-banner__primary-nav {
+  float:left;
+  margin:0 0 0 15px;
 }
 
 .b-banner__nav-item {
-  position:relative;
+  height:48px;
 }
 
-.b-banner__nav-item::before {
-  opacity:0;
-  position:absolute;
-  content:'';
-  background:#dd4814;
-  bottom:0;
-  left:0;
-  right:0;
-  height:2px;
-  transition: opacity 0.3s;
+.banner .nav-primary .b-banner__nav-item_active {
+  border-bottom:3px solid #e95420;
 }
 
-.b-banner__nav-item_active::before {
-  opacity:1;
+.b-banner__nav-item svg {
```

Comment:
To make these BEM styles cleaner it would be worth avoiding element names in selectors and add classes to them too (for example `b-banner__nav-item-icon instead` of `svg`). Same for `a` and `span` below (`nav-item-link`, `nav-item-label` maybe)?